src/Teams/Domain/MultipleAnswerQuestion.cs

```diff
@@ -32,5 +32,11 @@ public MultipleAnswerQuestionOption[] GetRightAnswers()
         {
             return Answers.Where(a => a.IsRight).ToArray();
         }
+
+        public void EditQuestion(string questionText, List<MultipleAnswerQuestionOption> Answers)
```

Comment:
The method parameter should not start with a capital. Rename the filed "answers" into "_answers" and put the "answers" name to the parameter.